hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java

```diff
@@ -292,6 +303,18 @@ private int handleMap(MemberStateImpl memberState, int count, Config config, Map
         return count;
     }
 
+    private int handleWan(MemberStateImpl memberState, int count, Map<String, LocalInstanceStats> wans,
+                          Set<String> longInstanceNames) {
+        for (Map.Entry<String, LocalInstanceStats> entry : wans.entrySet()) {
```

Comment:
there is `if (count >= maxVisibleInstanceCount)` check in the other handle methods. why don't you have it here?